query-engine/connector-test-kit-rs/query-engine-tests/tests/new/regressions/prisma_7010.rs

```diff
@@ -0,0 +1,75 @@
+use query_engine_tests::*;
+
+#[test_suite(schema(schema), only(MySql("8")))]
+mod special_id_values {
+    use indoc::indoc;
+
+    fn schema() -> String {
+        let prisma = indoc! {r#"
+        model BinTest {
+            id Bytes @id @default(dbgenerated("(uuid_to_bin(uuid()))")) @test.Binary(16)
+            name String
+        }
+        model StrTest {
+            id String @id @default(dbgenerated("(uuid())")) @test.Char(36)
```

Comment:
That is a great question, the answer will be "it depends"

- If the field is `String` and not length is specified, a `varchar(191)` is created by the migration engine (don't ask me why 191)
- If the field is `Binary` it will fail, in MySQL you cannot create BLOB fields without a size and different than with `String`, we don't have a default length for `Binary`.